docs/tutorials/nodes-tutorial.ipynb

```diff
@@ -0,0 +1,376 @@
+    "It is then up to each view to decide whether they are interested in the notification or not (usually, most views are interested in all notifications from parents).\n",
+    "If they are, they request data from their parent nodes, which in turn request data from their parents, and so on, until the request has reached the top of the graph.\n",
+    "\n",
+    "As a result, when the slider is dragged, the smoothing node `c` gets notified and tells the figure that a change has occured.\n",
+    "The figure tells node `c` that it wants updated data.\n",
+    "Node `c` asks nodes `a` and `b` for their data.\n",
+    "`a` returns the raw data, while `b` returns the integer value for the kernel size.\n",
+    "`c` then simply send the inputs to the `gaussian_filter` function, and forwards the result to the figure.\n",
+    "\n",
+    "## Multiple views\n",
+    "\n",
+    "To go one step further,\n",
+    "we now wish to add a second one-dimensional figure that will display the sum of the two-dimensional data along the vertical dimension.\n",
+    "On this figure, we would like to display both the original (unsmoothed) data, as well as the smoothed data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "686b3cdf-33b5-4e45-85b9-b2266211c0bc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "a = pp.input_node(da)\n",
+    "\n",
+    "slider = ipw.IntSlider(min=1, max=10)\n",
+    "b = pp.widget_node(slider)\n",
+    "\n",
+    "c = pp.node(gaussian_filter)(a, sigma=b)\n",
+    "\n",
+    "fig2d = pp.figure2d(c)\n",
+    "\n",
+    "# Sum the raw data along the vertical dimension\n",
+    "d = pp.node(sc.sum, dim='yy')(a)\n",
+    "# Sum the smoothed data along the vertical dimension\n",
+    "e = pp.node(sc.sum, dim='yy')(c)\n",
+    "# Give two nodes to a figure to display both on the same axes\n",
+    "fig1d = pp.figure1d(d, e)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "36c9ea09-8701-465c-beec-f55d371192ee",
+   "metadata": {},
+   "source": [
+    "We check the graph again to make sure that the one-dimensional figure has two inputs,\n",
+    "and that both are performing a sum along the `yy` dimension."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "81ae3b91-9622-48ca-bc90-f5cbce5ea426",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pp.show_graph(a)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d908a930-a1dc-4e7b-90f4-be8cb7e11d88",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ipw.VBox([slider, fig2d, fig1d])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4b595fcb-02b8-4fa3-be16-3512e65b2c6a",
+   "metadata": {},
+   "source": [
+    "Because the slider in only affecting the left part of the graph,\n",
+    "only the orange markers will update when we drag the slider."
```

Comment:
A bit risky to talk about 'the left part of the graph'. The layout might change if the algorithm or image size is tweaked.